packages/grafana-toolkit/README.md

```diff
@@ -1,55 +1,82 @@
-# Grafana Toolkit
 
-Make sure to run `yarn install` before trying anything!  Otherwise you may see unknown command grafana-toolkit and spend a while tracking that down.
+# grafana-toolkikt
+grafana-toolkit is a simple CLI that enables efficient Grafana extensions development
 
+## Why grafana-toolkit?
+Historically, creating Grafana extension was an excercise of reverse engineering and ceremeny around testing, developing and eventually building the plugin. We want to help our community to focus on the core value of their plugins rather than all the setup required to develop an extension.
 
+With grafana-toolkit we put a simple CLI in your hands that address common tasks performed when developing an extension:
+- `plugin:create`
+- `plugin:test`
+- `plugin:dev`
+- `plugin:build`
 
-## Internal development
-Typically plugins should be developed using the `@grafana/toolkit` import from npm.  However, when working on the toolkit, you may want to use the local version while underdevelopment.  This works, but is a little flakey.
+### Scaffolding extensions
 
-1. navigate to `packages/grafana-toolkit` and run `yarn link`.
-2. in your plugin, run `npx grafana-toolkit plugin:dev --yarnlink`
+__TODO:__ Implement the task...
 
-Step 2 will add all the same dependencies to your development plugin as the toolkit.  These are typically used from the node_modules folder
+Grafana supports 2 types of extensions: Panels and Datasources. Panels can be developed using either React or AngularJS(legacy).
+To scaffold new extension use `npx grafana-toolkit plugin:create`. This will guide you through the basic extension setup as well as create common directory structure and files from our extension templates. All of the templates comes with `grafana-toolkit` tasks available via yarn scripts: `yarn dev`, `yarn test` & `yarn build`.
 
 
-TODO: Experiment with [yalc](https://github.com/whitecolor/yalc) for linking packages
+### Developing extensions
 
+`plugin:dev`
```

Comment:
Most developers like to be able to copy-paste instructions. I'd recommend to write the full command, e.g. `grafana-toolkit plugin:dev` (assuming it doesn't need any more arguments or flags by default).